mappings/net/minecraft/util/registry/Registry.mapping

```diff
@@ -18,7 +18,7 @@ CLASS net/minecraft/class_2378 net/minecraft/util/registry/Registry
 	FIELD field_11156 SOUND_EVENT Lnet/minecraft/class_2378;
 	FIELD field_11157 CARVER Lnet/minecraft/class_2378;
 	FIELD field_11158 CUSTOM_STAT Lnet/minecraft/class_2378;
-	FIELD field_11159 STATUS_EFFECT Lnet/minecraft/class_2378;
+	FIELD field_11159 STATUS_EFFECT_TYPE Lnet/minecraft/class_2378;
```

Comment:
```suggestion
	FIELD field_11159 STATUS_EFFECT Lnet/minecraft/class_2378;
```
I think this is fine without the type suffix; its ID is `mob_effect` and we have other `*Type` registries without the suffix (`SCREEN_HANDLER`, `RULE_TEST` iirc).